flutter/simple_chat/lib/chat_room/chat_room_info_page.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:simple_chat/chat_room/chat_room.dart';
+import '../api_repository.dart';
+
+class ChatRoomInfoPage extends StatelessWidget {
+  const ChatRoomInfoPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final id = ModalRoute.of(context)!.settings.arguments as String;
+
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Channel Info'),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: BlocProvider.value(
```

Comment:
@oguibueno Do we need `ChatRoomsDataBloc` anywhere else except for this widget? If no, then I think we should use this instead of BlocProvider.value and remove the one from `main.dart`
```
BlocProvider<ChatRoomsDataBloc>(
   create: (context) => ChatRoomsDataBloc()..load .....
)
```